### ISSUE TITLE
Jenkins appears to be unable to process the "-" character when running tests.

### ISSUE THREAD

**Issue by coreyh3** [alaurenz/metrobike#95]
From http://nwebster.cs.washington.edu:8080/cse403/job/MetroBike/45/artifact/logcat.txt:

05-17 00:28:34.297 W/System.err(  883): Expected:
05-17 00:28:34.297 W/System.err(  883): Bicycling directions are in beta. Use caution ï¿½ This route may contain streets that aren't suited for bicycling.
05-17 00:28:34.307 W/System.err(  883): Actual:
05-17 00:28:34.307 W/System.err(  883): Bicycling directions are in beta. Use caution â€“ This route may contain streets that aren't suited for bicycling.
05-17 00:28:34.317 I/TestRunner(  883): failed: test_getWarningsTest(com.HuskySoft.metrobike.backend.test.RouteTest)

I'm not sure what the deal is but this runs completely fine locally.

**Comment by CoolCapri** [comment]
I tried running the test on my Mac using Android JUnit testing. It appeared that both getWarningsTest and toStringsTest failed

**Comment by coreyh3** [comment]
I'm not quite sure what the issue would be.  I do not have any local changes to RouteTest or Route in my repo. 
How are you running the unit tests?  I had to make a change to AndroidManifest.xml to get them running on the emulator.  I added the two highlighted lines below to make it work.
Line 34      <instrumentation android:targetPackage="com.HuskySoft.metrobike" android:name="android.test.InstrumentationTestRunner"></instrumentation>    <application        android:allowBackup="true"        android:icon="@drawable/ic_launcher"        android:label="@string/app_name"        android:theme="@style/AppTheme">        <uses-library android:name="android.test.runner" />        <activity

Corey

Date: Thu, 16 May 2013 18:09:26 -0700

**Comment by dutchscout** [comment]
I think I fixed it!

There were 2 causes:

1) The encoding for the file itself was a bit strange.  After fixing it a couple of times, I got the compiler to not waran about an unsupported character (the "-" in the test string).

2) The sample response JSON listed the "-" instead as "\u2013".  This is probably a unicode representatation of a similar character.  On our local systems, perhaps the character is seen as matching, while on Jenkins it is viewed as different.  I decided to replace the "\u2013" with "-" so our tests could continue.
